android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/ActionableButton.kt

```diff
@@ -99,7 +99,7 @@ fun ActionableButton(
               statusColor.copy(alpha = 0.08f)
             },
           contentColor = statusColor,
-          disabledBackgroundColor = DefaultColor.copy(alpha = 0.08f),
+          disabledBackgroundColor = statusColor.copy(alpha = 0.02f),
           disabledContentColor = DefaultColor,
```

Comment:
How about this disabled content color?

Reply:
The disabled content color had no effect on the button content so I opted not to interfere with it in the meantime.
Also if we are supposed to come up with the new config for the `CompoundText` and `ActionableButton` which of course is the right approach . I would consider we do just as you have suggested @ellykits